digitalpublicgoods/vaccineledger.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "VaccineLedger",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://gitlab.com/statwig-public/theledger/-/blob/development/LICENSE"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://gitlab.com/statwig-public/theledger",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger-_Use_Case___Functional_Features_.pdf",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/README.md"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "This data is stored on Blockchain and can be accessible through REST APIs.\nThe core idea of blockchain is collaboration based on shared data ownership, governance and  operations. The major difference from centralized system architectures is that peers,  collaborating in a network, can continue using their own systems and technology stack without  the need to integrate them in the classical way. Instead, blockchain provides a shared data  layer that allows these systems to interoperate on top and acts as the single truth. \n\nAs a result, the network has no single authority, which owns the data or controls the systems.  The decentralization of the ledger means a decentralization of power.\n\nLink to Evidence: https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger_-_Data_Confidentiality.pdf"
```

Comment:
I would suggest that links pointing to git repositories, use a direct link to a `sha` version, e.g.

https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger_-_Data_Confidentiality.pdf - a moving target
vs
https://gitlab.com/statwig-public/theledger/-/blob/050a19d2c92a2836bfb9483941e5219e5b90be62/docs/VaccineLedger_-_Data_Confidentiality.pdf  - a [permalink](https://en.wikipedia.org/wiki/Permalink)

Those links are _longer_ but are supposed to be the version of the document at the moment of the submission

Reply:
+1 @iperdomo